added website base

adelyntam opened the pull request:
has login/logout and registration pages

adelyntam committed:
added website base

has login/logout and registration pages

piklsr commented on app.py:
```diff
@@ -0,0 +1,119 @@
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+)
+db = SQLAlchemy(app)
+
```
#add this code here
@app.before_first_request
def create_tables():
    db.create_all()

adelyntam replied:
added

adelyntam committed:
fixed login/registration database

adelyntam committed:
fixed design and login page

users cannot access the login page once logged in. registration page matches the format of the login page. users can only access the register page through the login page.

piklsr reviewed (commented):
Error when registering a username that is already registered

adelyntam committed:
added code to get posts and show them on home page
